### DISCUSSION TITLE
Version 0.0.6: Improved memory system

### DISCUSSION THREAD

**Discussion by raspiduino** [raspiduino/arv32-opt#10] [Announcements]
Still need someone with real device to test

<hr /><em>This discussion was created from the release <a href='https://github.com/raspiduino/arv32-opt/releases/tag/'>Version 0.0.6: Improved memory system</a>.</em>

**Comment by zacharytomlinson** [comment]
You just need someone to compile and flash to a 328p? I can do that tonight if need be.

**Reply by raspiduino** [reply]
I want someone to run it (which also require a SD card and 8 hours)

**Reply by zacharytomlinson** [reply]
I have both. I will wire up an sd card connector to an old arduino tonight and let you know how it goes. Want me to post result pics back to this thread or somewhere else?

**Reply by raspiduino** [reply]
Thank you! You can post it right here!

**Reply by zacharytomlinson** [reply]
Hey quick update, I got everything initialized and connected properly except I kept getting the SD card initialization error. Long story short don't actually have a logic level shifter on hand and the adapter I was going to use, uses the SPI from an intermediary chip ( not just a logic level shifter but a totally separate ethernet related IC) in order to drive the 3.3v to the SD card. I would have to alter the source in order to make it work with this intermediary chip and that would be a lot more effort than just getting a separate SD card module with the  logic level shifter integrated. I ordered the module and it should be here in 1-2 days and I will update then. Sorry for the delay.

**Reply by raspiduino** [reply]
> Hey quick update, I got everything initialized and connected properly except I kept getting the SD card initialization error. Long story short don't actually have a logic level shifter on hand and the adapter I was going to use, uses the SPI from an intermediary chip ( not just a logic level shifter but a totally separate ethernet related IC) in order to drive the 3.3v to the SD card. I would have to alter the source in order to make it work with this intermediary chip and that would be a lot more effort than just getting a separate SD card module with the logic level shifter integrated. I ordered the module and it should be here in 1-2 days and I will update then. Sorry for the delay.

That's okay. Looking forward to your reply!
